modules/exploits/multi/http/spip_rce_form.rb

```diff
@@ -136,6 +131,17 @@ def execute_command(cmd, args = {})
     )
   end
 
+  def php_exec_cmd(encoded_payload)
+    dis = '$' + Rex::Text.rand_text_alpha(rand(4..7))
```

Comment:
We don't currently have a PHP generator for this sort of thing yet, but would you mind using the default Rex::RandomIdentifier? https://github.com/rapid7/rex-random_identifier/blob/master/lib/rex/random_identifier/generator.rb#L23C7-L23C39

The idea is to ensure that we're not accidentally creating variable names that are invalid in the context of PHP. We don't want to block this on waiting for a PHP definition, but we should add one in the future.